lib/cli/build.rs

```diff
@@ -1,4 +1,26 @@
+use chrono::prelude::*;
+use std::process::Command;
+
 pub fn main() {
-    println!("cargo:rerun-if-changed=build.rs");
-    println!("cargo:rerun-if-env-changed=WASMER_INSTALL_PREFIX");
```

Comment:
Right now the build script is run on every build, which is what I wanted because it sets the date of compilation on every single run, that's why I removed those lines.